src/app/features/collection/components/progress-bar-skeleton/progress-bar-skeleton.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="tw-flex tw-flex-col tw-items-center tw-w-full tw-p-1 tw-bg-zinc-50 tw-text-xs tw-text-black">
```

Comment:
to avoid extra login on collection level page - we can use that progress bar directly in progress bar component. It will required from us only one extra input - isLoading. In progress bar component we will display skeleton based on loading state.